python/test/unit/mesh/test_mesh_utils.py

```diff
@@ -249,7 +249,7 @@ def test_topology_partition(tempdir, shape, order):
 
     # Create (i) local topology object and (ii) IndexMap for cells, and
     # set cell-vertex topology
-    topology_local = cpp.mesh.Topology(layout.cell_type)
+    topology_local = cpp.mesh.Topology(cpp.MPI.comm_self, layout.cell_type)
```

Comment:
in this test file, you will need to update calls that use `cpp.MPI` due to #956

Reply:
Thanks for pointing out. I just rebased on master...